packages/tocco-ui/src/Upload/UploadInput.js

```diff
@@ -30,7 +30,6 @@ const UploadInput = props => {
           position={design.position.PREPEND}
         />
       </IconWrapper>
-      {props.text || 'drag and drop or click'}
```

Comment:
Is it possible to set this text as "title" attribute? otherwise we should delete it completely wich means here:
- stories
- tests
- form field factory
- remove text resource from nice2